aquacomputer_d5next.c

```diff
@@ -476,12 +507,37 @@ static int aqc_percent_to_pwm(u16 val)
 /* Converts to centi-percent */
 static int aqc_pwm_to_percent(long val)
 {
-	if (val < 0 || val > 255)
-		return -EINVAL;
-
 	return DIV_ROUND_CLOSEST(val * 100 * 100, 255);
 }
 
+/* Converts rpm to pwm */
+static int aquastream_rpm_to_pwm(long val)
+{
+	return DIV_ROUND_CLOSEST((val - AQUASTREAMXT_PUMP_MIN_RPM) * 255, 3000);
```

Comment:
`3000` is AQUASTREAMXT_PUMP_MIN_RPM?

Reply:
This `3000`  is the difference between the minimum and maximum rpm. 62d7872

Reply:
Thanks, I wasn't sure which one it was because they're equal.